views/package.html.twig

```diff
@@ -30,79 +30,76 @@
         <p class="lead">{{ package.highest.description }}</p>
         {% endif %}
 
-        <table class="table table-striped">
+        {% if package.highest.homepage %}
+        <div class="row">
+            <div class="col-lg-2 text-right"><strong>Homepage</strong></div>
+            <div class="col-lg-10"><a href="{{ package.highest.homepage }}">{{ package.highest.homepage }}</a></div>
```

Comment:
Yeah I just realized that the responsive alignment classes only exist in bootstrap 4 :<

Maybe use a dl + dd/dt approach instead?